index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Intuit Accounts - Sign Up</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <header class="login-wrapper">
+      <div class="login-navigation">
+        <div class="forgot-column">
+          <div id="forgot-text">Already have an Intuit account?</div>
+          <a href="#" id="recovery-link">I forgot my user ID or Password</a>
+        </div>
+        <div class="login-column">
+          <button id="login" class="login-submit" type="button">
+            <span class="lock-img"></span>
+            Sign In
+          </button>
+        </div>
+      </div>
+    </header>
+    <header class="main-header" id="main-header">
```

Comment:
- [ ] **[OPTIONAL]** You can also use `<nav>` tag instead of the `<header>` here

Reply:
Done.